### ISSUE TITLE
some question about code

### ISSUE THREAD

**Issue by tsingjinyun** [HantaoShu/DeepSEM#9]
in paper  formula 5 and 6 ,each gene is processed individual, but in code https://github.com/HantaoShu/DeepSEM/blob/master/src/Model.py#L94 , Fusion of genes from different groups. 
@HantaoShu

**Comment by HantaoShu** [comment]
Thanks for your interest in our work. We provide code with the GMVAE framework. And Line 94 in Model.py in to get the weight for the "K" Gaussian mixture distribution (K=1). Formulas 5 and 6 are referred to Line 150 (decoder) and Line 100 (encoder).  Any other questions are welcome.

**Comment by tsingjinyun** [comment]
logits, prob, y = self.qyx(x.squeeze(2), temperature) #Line131
mu, logvar = self.qzxy(x, y) # line 132
Line 132  will call Line 125 which will concat y from Line 131.
and Line 131 will call Line94, which will fuse gene infoamation, as show in  following picutre
![Picture1](https://user-images.githubusercontent.com/3369033/179198481-72db1dfe-ba99-4d04-8230-860f8dee73d9.png)

**Comment by HantaoShu** [comment]
Thanks for your reply, but you can find in Line 125 and also in your figure (repeat(1,shape[1],1), for each cell, the influence source from y are same for each gene. And in our default parameter (k=1), it's just add a bias for each gene.

**Comment by tsingjinyun** [comment]
y shape is 64*1 , which is calculate from x( 64*910), and i mean that y  have Fusing information between genes. your mean y is just add bias for each gene of 910?Thanks for your reply

**Comment by HantaoShu** [comment]
Yes, that is what I have stated, our model forces gene interaction which means updating gene embeddings based on embedding from the other genes.  Here, we add the same value for each gene which does not influence the main idea of our model. And we use GMVAE as the reviewer recommended especially for the embedding visualization section. One hypothesis is that the embedding of cells follows the gaussian mixture model, therefore, using GMVAE can improve the performance of embedding visualization. However, as the figure shows in the supplementary, we find adds more Gaussian doesn't significantly improve performance. Therefore, we use k=1 for default hyperparameters.

Any other questions are welcome.
